docker/Dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG ROS_VERSION=jazzy
+
+FROM osrf/ros:${ROS_VERSION}-desktop
+
+SHELL ["/bin/bash", "-c"]
+
+# Username
+ENV user=vizanti
+
+# Update and build the vizanti package
+RUN apt-get update
+
+# Need openssh-client for git to work properly, can be removed when rosbridge has binaries properly updated.
+RUN apt-get install openssh-client -y
```

Comment:
If we add the RMW arg, we also need to install ros-${ROS_VERSION}-rmw-cyclonedds-cpp here and ros-${ROS_VERSION}-rmw-fastrtps-dynamic-cpp for RWS support.